src/core/webflow.ts

```diff
@@ -22,8 +22,45 @@ export interface Options {
   token?: string;
   version?: string;
   headers?: Record<string, string>;
+  beta?: boolean;
 }
 
+type MethodNames =
+  | "info"
+  | "authenticatedUser"
+  | "sites"
+  | "site"
+  | "publishSite"
+  | "domains"
+  | "collections"
+  | "collection"
+  | "items"
+  | "item"
+  | "createItem"
+  | "updateItem"
+  | "patchItem"
+  | "removeItem"
+  | "deleteItems"
+  | "publishItems";
+
+export const SCOPES_ARRAY = [
```

Comment:
We may also want to add scopes for `users:read`, `users:write`, `ecommerce:read` and `ecommerce:write`